programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,6 @@
+function solution(s) {
+  let answer = "";
+  const stringArr = s.split("").sort().reverse();
+  answer = stringArr.join("");
+  return answer;
```

Comment:
```suggestion
  return s.split("").sort().reverse().join("");
```
조금 더 간결하죠? 그래도 잘 해결하셨어요 ㅎㅎ